### ISSUE TITLE
Release of the dataset.

### ISSUE THREAD

**Issue by Emiya-syw** [T-SciQ/T-SciQ#2]
Excellent work! Could you offer the dataset introduced in this paper? It's a little expensive for me to generate all of them. Thanks a lot!

**Comment by BAOOOOOM** [comment]
Indeed, if the QA-CoT dataset and the QA-PCoT dataset could be made public, it would better facilitate further research by subsequent scholars.
